### DISCUSSION TITLE
🚀 Feature request - add new options to the HTTP Headers plugin

### DISCUSSION THREAD

**Discussion by 5BYFIVE** [ExpressionEngine/ExpressionEngine#4315] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
Client is running security scans on an EE website. There are a couple http headers missing. These headers aren't currently an option on the HTTP Headers plugin.

**Describe the solution you'd like**
It would be nice to get a couple new header options added to the HTTP Headers plugin to use...“Content-Security-Policy” and “Content-Security-Policy-Report-Only”.

**Describe alternatives you've considered**
Setting up headers via server settings

**Teachability, Documentation, Adoption, Migration Strategy**
Example value...
`default-src 'self'; script-src 'report-sample' 'self' https://cdn.jsdelivr.net/npm/@fancyapps/ui@4.0/dist/fancybox.umd.js https://modelviewer.dev/node_modules/@google/model-viewer/dist/model-viewer.min.js https://unpkg.com/alpinejs@3.9.0/dist/cdn.min.js; style-src 'report-sample' 'self' https://cdn.jsdelivr.net https://p.typekit.net https://use.typekit.net; object-src 'none'; base-uri 'self'; connect-src 'self' blob: data:; font-src 'self' https://use.typekit.net; frame-src 'self' https://online.fliphtml5.com; img-src 'self'; manifest-src 'self'; media-src 'self'; report-uri https://663bbd01da2354f69a82bb78.endpoint.csper.io/?v=2; worker-src 'none';`

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by bryannielsen** [comment]
This should be addressed when 7.4.10 is released. Thanks to @dougblackjr's PR - https://github.com/ExpressionEngine/ExpressionEngine/pull/4316
